behave/features/foxadmin_users.feature

```diff
@@ -8,22 +8,34 @@ Background: Log in to the Fox admin as a user within the super users group
 Scenario: Navigate to users authentication page and assign user staff status
 Given I select the link "Users"
 When I select a non-staff user from the list
-And I am taken to the user's details page
+And I am taken to the non-staff user's details page
 And I select Staff status under permissions
 And I select save
 Then the users details are saved and I am taken back to the list of users
 And I select the link "Log out"
 
-#Journey P11 Child 2, Child 3 Create an operator user and login (LGA-1864, LGA-1865)
+#Journey P11 Child 2, Child 3, Child 4, Create an operator user, login CHS with new user then delete user
+# (LGA-1864, LGA-1865, LGA-1866)
 @fox-create-user
 Scenario: Create an operator user
 Given I select the link "Operators"
 And I choose to "Add operator"
 And I am taken to the "Add operator" page located on "/admin/call_centre/operator/add/"
 And I create a new operator user
-And I select 'Is active’
+And I select 'Is active'
 And I choose to "save"
 Then the new operator user is created
 And I am taken to the list of operators page
+#Child 3
 And I select the link "Log out"
-And that I am logged in as "FOX_ADMIN_NEW_USER"
\ No newline at end of file
+And that I am logged in as "FOX_ADMIN_NEW_USER"
```

Comment:
This is a bit confusing as you are logging into the CHS but as a new fox admin user
What do you think of renaming that parameter to something like 'NEWLY_CREATED_OPERATOR' or something like that?

Reply:
Yeah I can see that. I've made the changes